torch/jit/__init__.py

```diff
@@ -639,6 +639,8 @@ def __getattr__(self, attr):
 
 
 def _try_get_dispatched_fn(fn):
+    if not callable(fn):
```

Comment:
What is this doing ?

Reply:
It's just there for better error messages, if something that Python can't create a weak reference to then `.get()` throws an error instead of a more informative error like `Python value ... cannot be used in script`